tide_site.module

```diff
@@ -678,3 +681,14 @@ function tide_site_form_views_exposed_form_alter(&$form, FormStateInterface $for
     }
   }
 }
+
+/**
+ * Implements hook_ENTITY_TYPE_access().
+ */
+function tide_site_taxonomy_term_access(EntityInterface $entity, $operation, AccountInterface $account) {
+  if ($entity->bundle() == 'sites' && $operation == 'delete') {
+    if (!in_array('administrator', $account->getRoles())) {
```

Comment:
If we can't revoke `Administer vocabularies and terms` maybe we should create a new permission in `tide_site` and check for that here rather than not being an administrator?

Reply:
yeah, we can create a new permission. 
here is the plan.
1. we revoke `Administer vocabularies and terms` permission
2. add new custom permission eg: `Add new vocabulary`.
3. assign `Add new vocabulary` to users who can create a new vocabulary.
3. we use build-in permission `Sites: Delete terms`
4. an update hook is needed.

Reply:
I think we don't need this PR. tide_core provides permission settings.